store.go

```diff
@@ -2251,6 +2255,7 @@ func (s *store) Shutdown(force bool) ([]string, error) {
 		s.graphLock.Touch()
 		modified = true
 	}
+	mount.Unmount(s.RunRoot())
```

Comment:
Move this so that it's only called if `err` is `nil`, along with `s.graphdriver.Cleanup()`.